src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -176,6 +176,22 @@ The API used to retrieve the value is:
 * [GCGenerationInfo.FragmentationAfterBytes Property](https://docs.microsoft.com/dotnet/api/system.gcgenerationinfo.fragmentationafterbytes)
   Gets the fragmentation in bytes on exit from the reported collection.
 
+#### process.runtime.dotnet.**gc.duration**
```

Comment:
Please add a hint that this is availably in .NET 7+ only

Reply:
I have a note in section:

```
> **Note**
> This metric is only available when targeting .NET 7 or later."
```